api/Sources/Api/PairQL/Dashboard/Pairs/SaveConferenceEmail.swift

```diff
@@ -0,0 +1,35 @@
+import PairQL
+import Vapor
+
+struct SaveConferenceEmail: Pair {
+  static var auth: ClientAuth = .none
+
+  struct Input: PairInput {
+    enum Source: String, Codable {
+      case workshop
+      case booth
+    }
+
+    var email: String
+    var source: Source
+  }
+}
+
+// resolver
+
+extension SaveConferenceEmail: Resolver {
+  static func resolve(with input: Input, in context: Context) async throws -> Output {
+    let detail = "SaveConferenceEmail: \(input.email), source: \(input.source)"
+    _ = try? await InterestingEvent.create(.init(
```

Comment:
quick and easy, leverage an existing database table, since we may only get a few of these.